releasePlan.cfg

```diff
@@ -12,4 +12,4 @@ jdk8uGA="jdk8u442-ga"
 jdk11uGA="jdk-11.0.26-ga"
 jdk17uGA="jdk-17.0.14-ga"
 jdk21uGA="jdk-21.0.6-ga"
-jdk23uGA="jdk-23.0.2-ga"
+jdk24GA="jdk-24-dryrun-ga"
```

Comment:
Do we drop 23.0.2 as we move into 24?

Reply:
Add it back. Drop 23 on April 2025 release.